sudoers/init.sls

```diff
@@ -16,6 +16,10 @@ sudo:
     - source: salt://sudoers/files/sudoers
     - context:
         included: False
+{%- if salt.grains.get('oscodename') == 'trusty' %}
```

Comment:
I don't think it's actually the distribution that affects whether the `json` or `tojson` filter is appropriate.  I thought it was the salt version, but that's actually consistent across trusty and bionic.  Is it the python version that salt is running under?